include/es/pooled_resource.h

```diff
@@ -9,20 +9,24 @@
 #include <boost/foreach.hpp>
 #include <map>
 #include <vector>
+#include <signal.h>
 
-/* 
+/*
  * this class pools a single resource type
  * providing the items from the pool when available
  * and NULL if all the pool items have been claimed
  * it does no allocation for you
  */
 template <class T>
 class pooled_resource {
-    public:
+ public:
```

Comment:
this (the indenting) was just sloppiness... it's just where my editor auto-tabbed the public... not important.  I think it just auto-tabbed when I was messing around with the destructor.